pkg/pool-weighted/contracts/lib/CircuitBreakerLib.sol

```diff
@@ -78,16 +78,16 @@ library CircuitBreakerLib {
     // the circuit breaker was set, these stored values can still be used, avoiding a heavy computation.
```

Comment:
This is where we should say e.g. 'we store a reference weight, so that if the current weight equals the reference weight we can skip the expensive computation of the current ratios